docs/pages/user-docs/subpages.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Subpages
+parent: User Guide
+nav_order: 640
+---
+# :diamond_shape_with_a_dot_inside: Subpages
+## :trident: Organigram 
+
+This subpage offers you a visualization of our organizational structure. By scrolling down the page you find different work units which can be expanded by clicking on the arrow far to the right. 
+## :door:  Rooms 
+When you open the room subpage you can directly open our internal **booking tool** and check for available facilities as in Outlook. If this doesn't work, the function is disabled by your company
```

Comment:
wow the thing with the booking tool is cool stuff. Is this also publicly available? Or is this only MMS internal feature? 
By the way @DanielHabenicht && @mschwrdtnr  this is a cool feature :)

Reply:
It's just a link to the internal booking tool. Every company can provide an bookingtool-link which then will be rendered there.

Reply:
> Warning: Offtopic!

ah okay. Hm maybe we should think about a little bit more syntax for this url. So we can fill query parameter with the room information. Just an Idea. maybe I create an Issue.

Reply:
Well if there is a tool, that supports that we can talk about it.